pandas/tests/io/test_html.py

```diff
@@ -1233,8 +1233,8 @@ class ErrorThread(threading.Thread):
             def run(self):
                 try:
                     super().run()
-                except Exception as e:
-                    self.err = e
+                except Exception as err:
```

Comment:
I think you might have mentioned in the past but is there a flake8 rule for single character variable names? Worth turning on?

Reply:
IIRC there is a rule that excludes lower-case "l", upper-case "I", and upper-case "O" as ambiguous.  My complaint about the more general case is that these are hard to grep for.